src/pages/token-metadata/pnfts.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Programmable NFTs
-metaTitle: Programmable NFTs | Token Metadata
+title: Programmable NFTs (pNFTS)
+metaTitle: Programmable NFTs (pNFTS) | Token Metadata
 description: Learn more about Programmable NFTs (a.k.a. pNFTs) on Token Metadata
 ---
 
-As mentioned in the [overview page](/token-metadata#pnfts), Programmable NFTs (PNFTs) are a new asset standard that allows creators to define custom rules on specific operations and delegate more granularly to third-party authorities. {% .lead %}
+As mentioned in the [overview page](/token-metadata#pnfts), Programmable NFTs (pNFTs) are a new asset standard that allows creators to define custom rules on specific operations and delegate more granularly to third-party authorities. {% .lead %}
 
 ## No More Bypassing Token Metadata
 
-Because the Token Metadata program is built on top of the SPL Token program, any owner or spl-token delegate can interact with the SPL Token program directly and bypass the Token Metadata program on vital operations like transferring and burning. Whilst this creates a nice composibility pattern between programs it also means the Token Metadata program cannot enforce any rules on behalf of the creators.
+Because the Token Metadata program is built on top of the SPL Token program, any owner or spl-token delegate can interact with the SPL Token program directly and bypass the Token Metadata program on vital operations like transferring and burning. Whilst this creates a nice composability pattern between programs it also means the Token Metadata program cannot enforce any rules on behalf of the creators.
 
-A good example of why this can be problematic is that Token Metadata cannot enforce secondary sales royalties. Even though the royalty percentage is stored on the **Metadata** account, it is up to the user or program that performs the transfer to decide whether they want to honour it or not. We talk more about this and how PNFTs fix this issue [in a section below](#use-case-royalty-enforcement).
+A good example of why this can be problematic is that Token Metadata cannot enforce secondary sales royalties. Even though the royalty percentage is stored on the **Metadata** account, it is up to the user or program that performs the transfer to decide whether they want to honour it or not. We talk more about this and how pNFTs fix this issue [in a section below](#use-case-royalty-enforcement).
 
-Programmable NFTs are introduced to solve this issue in a flexible way that **allows creators to customise the authorization layer** of their assets.
+Programmable NFTs are introduced to solve this issue in a flexible way that **allows creators to customize the authorization layer** of their assets.
```

Comment:
Interesting detail choice we've never discussed afaik.   But since have chosen to change "customize" to USA English.  So then should "honour" be changed to  "honor"?

Reply:
Yeah it was just one I spotted on the fly. I should probably go through with a find and replace on english to american english as I don't think we've ever done a deep dive on that. I know I'm one to get them mixed up occasionally.